sentinel-core/src/main/java/com/alibaba/csp/sentinel/node/metric/jmx/MetricBeanTimerListener.java

```diff
@@ -40,11 +40,12 @@ public void run() {
     
     private List<MetricNode> getLastMetrics(ClusterNode node) {
         final long currentTime = TimeUtil.currentTimeMillis();
-        final long minTime = currentTime - currentTime % 1000;
+        final long maxTime = currentTime - currentTime % 1000;
+        final long minTime = maxTime - 1000;
         return node.rawMetricsInMin(new Predicate<Long>() {
```

Comment:
@cdfive I change the poolSize of SCHEDULER into 1, so the two listener will run one by one in thread-safe.  And I use the method  rawMetricsInMin instead of metrics to get the metrics in last sencod so that the JMX will be same as  metrics in file.

Reply:
@cdfive any idea about this?

Reply:
@brothelul Nice work! I can get the expected result in jconsole now.